code/game/jobs/jobs.dm

```diff
@@ -183,3 +183,19 @@ var/list/whitelisted_positions = list(
 
 	return titles
 
+var/global/list/exp_jobsmap = list(
+	EXP_TYPE_LIVING = list(), // all living mobs
+	EXP_TYPE_CREW = list(titles = command_positions + engineering_positions + medical_positions + science_positions + support_positions + supply_positions + security_positions + civilian_positions + whitelisted_positions), // crew positions

```

Comment:
Oh, this doesn't include the nonhuman positions. Might want to `|` these lists together to eliminate duplicate titles, though, in case anyone ever tries to iterate over them for some reason.